src/lib/types/Assistant.ts

```diff
@@ -14,4 +14,9 @@ export interface Assistant extends Timestamps {
 	preprompt: string;
 	userCount?: number;
 	featured?: boolean;
+	rag?: {
+		allowAll: boolean;
+		allowList: string[];
+		links: string[];
```

Comment:
```suggestion
		allowAllDomans: boolean;
		allowedDomains: string[];
		allowedLinks: string[];
```
the naming could be more specific

Reply:
agreed, will change it everywhere